docs/config.rst

```diff
@@ -1511,6 +1650,7 @@ Valid preset color variables are:
 
 .. attribute:: LP_COLORMAP
    :type: array<string>
+   :value: ( "" $GREEN $BOLD_GREEN $YELLOW $BOLD_YELLOW $RED $BOLD_RED $WARN_RED $CRIT_RED $DANGER_RED )
```

Comment:
>  Line too long (104/80) (line-too-long)

You can wrap the line and it still works:
```suggestion
   :value: ( "" $GREEN $BOLD_GREEN $YELLOW $BOLD_YELLOW $RED
             $BOLD_RED $WARN_RED $CRIT_RED $DANGER_RED )
```

There is also a few lines below a definition of the default as well; that should be removed.